### ISSUE TITLE
Possible issue with Visa Cal?

### ISSUE THREAD

**Issue by esakal** [eshaham/israeli-bank-scrapers#602]
Hi All,
Something is wrong with visa cal, today they stopped sending transaction type thus for me at least the scraper fails. 

I'm going to wait till tomorrow to see if this is a bug or not on their side.

![CleanShot 2021-08-31 at 21 00 40@2x](https://user-images.githubusercontent.com/4751797/131553072-cdace715-1792-4ac2-9b54-39dc22634527.jpg)

**Comment by esakal** [comment]
@brafdlog yes it is the same issue. You can try my proposal in #603, I'm working with it since then and beside not having memo it seems fine.

The options are either to use my proposal or to re-write the scraper from scratch using browser scrapping approach.

@eshaham please advice

**Comment by tpseven-777** [comment]
Got the very same issue. How do I get the patch before a version is released?

**Comment by esakal** [comment]
@tpseven-777 You can have the dependency from the PR directly. see https://stackoverflow.com/questions/33181297/npm-install-from-github-pull-request

I don't have the ability to merge my own PR so we need @eshaham assistance. But regardless the PR seems to be partial because although it fixes the null issue, it doesn't resolve the missing memo and potentially / maybe issue with pending transactions.

It seems that the proper solution would be to re-write the scraper from scratch using puppeteer, unless someone is familiar with another api that we can use directly.

**Comment by RoiNsec** [comment]
Hi
Is there an expectation when this will be resolved?

**Comment by tpseven-777** [comment]
@esakal I've just tried installing this PR by with the following:
`  821  npm install --save-dev  eshaham/israeli-bank-scrapers#pull/603/head`
`  822  npm install --save-dev  israeli-bank-scrapers/israeli-bank-scrapers#pull/603/head`
`  823  npm -i --save-dev  israeli-bank-scrapers/israeli-bank-scrapers#pull/603/head`
`  824  npm -i --save-dev  israeli-bank-scrapers/israeli-bank-scrapers\#pull/603/head`
`  825  npm install --save-dev  israeli-bank-scrapers/israeli-bank-scrapers\#pull/603/head`
`  826  npm install --save-dev  eshaham/israeli-bank-scrapers\#pull/603/head`
`  827  npm install git+https://github.com/`
`  828  npm install git+https://github.com/eshaham/israeli-bank-scrapers\#fix-visa-cal-types`

But it doesn't seem like this was working, I've got error codes on those trials.
I'm probably missing something, can you tell me what it is?

**Comment by esakal** [comment]
@tpseven-777 my bad, as this is compiled library you cannot install directly from the pull request. What I'm doing for now in my project is:
1. pull the PR locally
2. change the package name in `package.json` - postfix it by `-hotfix.1`. the value will be `1.0.4-hotfix.1`.
3. run `npm build`
4. run `npm pack`
5. a file named `israeli-bank-scrapers-1.0.4-hotfix.1.tgz` will be created.
6. copy the file to your project under folder `lib`
7. run `npm I file:lib/israeli-bank-scrapers-1.0.4-hotfix.1.tgz` to install from the local file
8. don't forget to commit the file with your project.

once the official version will be deploy, you will be able to just install from NPM registry.

Note that if you do this trick multiple time you must increase the hotfix number. for example I'm with `hotfix.3`

![CleanShot 2021-10-09 at 19 59 16](https://user-images.githubusercontent.com/4751797/136667576-592e007a-951c-44bb-af3b-f0b5601fef1f.jpg)

Feel free to ask for clarifications if needed
